releaser/composite/action.yml

```diff
@@ -48,7 +48,7 @@ runs:
     run: pip install PyGithub --progress-bar off
 
   - shell: bash
-    run: ${{ github.action_path }}/../releaser.py
+    run: '''${{ github.action_path }}/../releaser.py'''
```

Comment:
Why not single quote-double quote -> `'"xxx"'`?

Reply:
I guess we want the path to be handled as a literal string, to avoid the shell processing it. The replacement of `${{ github.action_path }}` is done by the runner before writing the block to a shell file.

Reply:
Yes.  A double-quoted string in the shell would still interpret backslashes as escapes, so we need single quotes to go to the shell.  I am not as familiar with YAML, so I bet there are other options that would work to accomplish that goal.